### ISSUE TITLE
Úpravy Workflow

### ISSUE THREAD

**Issue by SykoraLukas** [proarc/proarc-client#655]
Na schuzce 2024.12.10 dohodnotu následující změny:
- [x] Obrazovka Úkoly přidat sloupec signatura
- [x] Držet filtry u záměru a úkolu při přechodu mezi obrazovkami
- [x] Do záhlaví sloupců přidat defaultní hodnoty podle kterých se bude hledat přesná shoda
- [x] U obrazovky úkol a záměru zobrazovat místo ownerId hodnotu ownerName. Hledá se podle ownerId ale uživateli je zobrazena hodnota ownerName
- [x] U sloupce, podle kterého se filtruje je zvýraznit záhlaví sloupce na font-weight: 900

**Comment by nezbedova** [comment]
@SykoraLukas ještě chybí

Při zakládání nového záměru přejmenovat název u vyskakovacího okna. Změnit slovo _objekt_ na _záměr_. Slovo _Nový_ zůstane.

![obrazek](https://github.com/user-attachments/assets/0c6822f2-a0d9-4a5a-8d0c-1fed5a1c3ace)

**Comment by SmejkalovaAnna** [comment]
@SykoraLukas Lukáši prosím tě, mohl bys prosím přidat k uuid ještě funkci a ikonku Zkopírovat do schánky? Šlo by to? 
![obrazek](https://github.com/user-attachments/assets/c50772ae-2919-48a9-9e3d-85dfedf01dc9)

**Comment by SykoraLukas** [comment]
@albertoh tohle je spis na tebe

**Comment by SykoraLukas** [comment]
Na API sloupec přidán, je potřeba nyní přidat i na klientovi.

**Comment by albertoh** [comment]
> Při zakládání nového záměru přejmenovat název u vyskakovacího okna. Změnit slovo objekt na záměr. Slovo Nový zůstane.

 Hotovo

>  přidat k uuid ještě funkci a ikonku Zkopírovat do schánky?

Hotovo

**Comment by nezbedova** [comment]
Vše, co je označeno jako hotové, jsem otestovala dnes na 

Klient: Verze v2.3.3-dev (07.01.2025)Jádro: Verze 4.2.4-SNAPSHOT - Build 20250107154000

Vše funguje. Děkujeme.

Zbývá:

- [ ] Do záhlaví sloupců přidat defaultní hodnoty podle kterých se bude hledat přesná shoda

**Comment by albertoh** [comment]
> Do záhlaví sloupců přidat defaultní hodnoty podle kterých se bude hledat přesná shoda

Pridano

**Comment by SykoraLukas** [comment]
U těchto sloupců se řadit nedá, protože se uživatelům sice zobrazí nějaký text, ale v databázi je uložen klíč, který neodpovídá tomu, co vidí uživatel. Proto se to nedá seřadit abecedně.

**Comment by pkudela** [comment]
Upvail jsem ve workdlow ty filtry, ktere jsou v hlavicce tabulky, prosim otestujte, jeslti takto vyhovuje.

<img width="582" alt="Image" src="https://github.com/user-attachments/assets/ba17efc3-7f91-411d-8788-7ed90f7c30f6" />

<img width="614" alt="Image" src="https://github.com/user-attachments/assets/aabb18c5-fb5a-448d-97ae-85d625a19764" />

**Comment by SmejkalovaAnna** [comment]
@pkudela Petře? Co se děje? Mě to hází z roscestníku sem: 

![Image](https://github.com/user-attachments/assets/d3e8b060-cfce-4e3b-aca0-a01c7265cd91)

**Comment by pkudela** [comment]
> [@pkudela](https://github.com/pkudela) Petře? Co se děje? Mě to hází z rozcestníku sem:
> 
> ![Image](https://github.com/user-attachments/assets/d3e8b060-cfce-4e3b-aca0-a01c7265cd91)

To jsem zrovna nasazoval, uz by to melo byt ok.

**Comment by SmejkalovaAnna** [comment]
@pkudela Promiň, už dobrý...

**Comment by SmejkalovaAnna** [comment]
@pkudela prošla jsem to a přijde mi to po všech směrech funkční.... ještě radši počkám na Martinu @nezbedova , co řekne. Mě to přijde dobrý.

**Comment by pkudela** [comment]
> [@pkudela](https://github.com/pkudela) prošla jsem to a přijde mi to po všech směrech funkční.... ještě radši počkám na Martinu [@nezbedova](https://github.com/nezbedova) , co řekne. Mě to přijde dobrý.

ano, funkcnost byla uz pred tim akorat to bylo cele rozhozene, nebylo to vubec odesignovane

**Comment by nezbedova** [comment]
@pkudela Ještě, prosím, sloupec Zařízení (ideálně s roletkou a vybrat si) a sloupec Poznámka. Děkuji

**Comment by albertoh** [comment]
Zařízení hotovo (je nutne zmazat cache)
Poznámka: nerozumim problem

**Comment by nezbedova** [comment]
@albertoh sloupec Zařízení funguje.

Ještě nefunguje sloupec "Poznámka". Netřídí podle abecedy ani nehledá.

![Image](https://github.com/user-attachments/assets/58fa59c4-750e-4e56-bebc-8b9d31145a57)

**Comment by SykoraLukas** [comment]
Filtrování podle poznámky funguje. Řazení nefunguje, protože je špatně nastaven PostgreSQL (nebude zatím řešeno).

**Comment by nezbedova** [comment]
Funguje. děkuji.

**Comment by tumovam** [comment]
Issue už je hrozně nepřehledné. Zavírám a příp, založím issues s konkrétními bugy.
